### ISSUE TITLE
Any plans to make this work with hoisted lerna projects?

### ISSUE THREAD

**Issue by danielantelo** [tnobody/lerna-audit#11]
Currently struggling to run an auto `npm audit fix` on our lerna project which is configured to hoist. Have tried `npx lerna-audit` but it seems to look for lock files in each package instead of following the lerna hoist conf route.

**Comment by svettwer** [comment]
Hi :wave:!

Thx for your input. Currently hoisted projects are not supported but would be a cool improvement. AFAIK there are no plans to add support for hoisted lerna projects in the near future, so a PR would be highly appreciated. 

BR
Sven

**Comment by IgnorantSapient** [comment]
Hey @svettwer is this project still being actively maintained, I'd love to coordinate with you and make lerna-audit and make it work with hosted projects.

I am more than happy to send you PR.

**Comment by svettwer** [comment]
Hey @123aswin123! :wave:

@tnobody and I are sharing the maintenance of this project. PRs are always welcome and will be reviewed, merged and released at best effort. If you'd like to propose a new feature, I'll be happy to review and help with questions. 😁👍

**Comment by IgnorantSapient** [comment]
Okay, so I'd like to send a PR for lerna-audit to work with hoisted lerna projects, do I raise a new issue for tracking this? or can we discuss it in this issue itself?

Thanks for your help @svettwer

**Comment by svettwer** [comment]
As this issue is about hoisted project support, I'd just reopen it, so that we can discuss here. :+1:

**Comment by utsavkapoor** [comment]
Any updates on this enhancement. I wanted to use lerna-audit package for my hoisted repo

**Comment by svettwer** [comment]
Nope, no update currently. But we're always happy to receive PRs. :hugs:
